whisper/whisperv6/whisper.go

```diff
@@ -591,7 +638,10 @@ func (wh *Whisper) add(envelope *Envelope) (bool, error) {
 
 	if envelope.PoW() < wh.MinPow() {
 		log.Debug("envelope with low PoW dropped", "PoW", envelope.PoW(), "hash", envelope.Hash().Hex())
-		return false, nil // drop envelope without error
+		return false, nil // drop envelope without error for now
+
+		// after the Status message will include the PoW requirement, it should return an error here:
```

Comment:
once the status message includes the PoW requirement, an error should be returned here